codemon/codemon.py

```diff
@@ -19,11 +19,8 @@ def main():
       if arg == "init":
         if sys.argv[countArg] == '-n':
           file = sys.argv[countArg+1]
-          path = '.'
-          f = open(path + '/' + file + '.cpp',"w+")
           template = template_cpp()
-          f.write(template)
-          f.close()
+          write_to_file(f'{file}.cpp', template)
           print(colored.yellow("Created "+file+'.cpp'))
```

Comment:
Can you rewrite this line as :
```
print(colored.yellow("Created {}.cpp".format(file))
```

Reply:
I think using an f-string would be better and improve the readability too.

Reply:
Okay. Go ahead